django/getting-started/index.html.md

```diff
@@ -187,7 +139,7 @@ By default, Django is configured for local development. The [How to Deploy Djang
 
 However, for demonstration purposes, we can take some shortcuts.
 
-First, in the [`hello_django/settings.py`](https://github.com/fly-apps/hello-django-fly/blob/main/hello_django/settings.py) file update the `ALLOWED_HOSTS` configuration to accept all hosts:
+First, in the [`hello_django/settings.py`](https://github.com/fly-apps/hello-django/blob/main/hello_django/settings.py) file update the `ALLOWED_HOSTS` configuration to accept all hosts:
 
 ```python
 # hello_django/settings.py
```

Comment:
@felixxm could we also update the `<-- Updated!` here to `← Updated!`/`← Added!` to keep consistency on the `←` (same way you did [here](https://github.com/superfly/docs/pull/1202/files#diff-2c1754b2e47e9eafdaa9fbe02f6aaf42f39ba3c34ad54e240b5151a712b32194R110))